lib/src/features/line_renderer.dart

```diff
@@ -61,7 +69,48 @@ class LineRenderer extends FeatureRenderer {
         continue;
       }
 
-      context.canvas.drawPath(line, effectivePaint);
+      // do we need a dashed line?
+      if (style.lineLayout!.dashArray.length >= 2) {
+        final dashedLine = dashPath(line, CircularIntervalList(dashLengths));
+        context.canvas.drawPath(dashedLine, effectivePaint);
+      } else {
+        context.canvas.drawPath(line, effectivePaint);
+      }
+    }
+  }
+
+  // convert a path into a dashed path with given intervals
+  Path dashPath(Path source, CircularIntervalList<num> dashArray) {
```

Comment:
Does this show up in the profiler? I wonder if we should store the results on the `TileFeature` so that we don't have to recalculate the path every time we render a tile.

Reply:
Please consider moving `dashPath` out of `LineRenderer` since drawing a line and making a path dashed are two separate concerns.

Reply:
-> new conv. below